website/landing/components/AdvancedUsage/AdvancedUsage.tsx

```diff
@@ -0,0 +1,122 @@
+import { useSpring, useTransform, useViewportScroll } from "framer-motion";
+import React, { useLayoutEffect, useRef, useState } from "react";
+
+// import config from "../../website.config.json";
+import {
+  List,
+  ListItem,
+  SectionContainer,
+  SectionHeader,
+  SectionTitle,
+  SectionWrapper,
+} from "../common";
+import { ParallaxLogo } from "../common/ParallaxLogo";
+
+import { UsageExample } from "./UsageExample";
+
+const SPRING_OPTIONS = { stiffness: 50, damping: 10 };
+
+export const AdvancedUsage: React.FC = () => {
+  // TODO: investigate why the app isn't able to find the existing
+  // content from the website config.
```

Comment:
It gets the content fine when running locally (or serving a prod build) but it gets stuck on the following error on the editor v2.

<img width="597" alt="advanced_usage_error" src="https://user-images.githubusercontent.com/24959348/142553268-5db8177d-b07d-4f41-9d8a-146be6968d5a.png">

I guess it might be content rendering before the config gets set.
To solve it, I've tried assigning the content to the state and updating it with an effect.

```tsx
const CONTENT_KEY = "advancedUsage";

const [content, setContent] = useState(config[CONTENT_KEY] || {});

useEffect(() => { setContent(config[CONTENT_KEY] || {}) }, [config]);
```

This fixes part of it and shows the examples but doesn't get the title value, so I hard coded the content here with an intent to investigate it later.